build/cli/src/lib.rs

```diff
@@ -433,9 +433,9 @@ impl Processor {
             arg::backend::Command::CiCheck {} => {
                 let config = enso_build::engine::BuildConfigurationFlags {
                     build_benchmarks: true,
-                    // Windows is not yet supported for the native runner.
-                    build_native_runner: enso_build::ci::big_memory_machine()
```

Comment:
I am wondering why we `build_native_runner` on [Mac Arm](https://github.com/enso-org/enso/pull/10329/files#r1649552868)? Probably the Mac Arm classifies as `big_memory_machine`...